From 0b221618f85f3e2f00115c6798bfbce7da28f513 Mon Sep 17 00:00:00 2001
From: Tanmay Jagdale <tanmayj@broadcom.com>
Date: Fri, 5 Dec 2014 19:39:40 +0530
Subject: smsc: added smsc entries in dts files

Added SMSC entries in the device tree files for XLP2XX and XLP3XX
[Based on SDK 3.2]
Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
Signed-off-by: Nam Ninh <nam.ninh@windriver.com>

diff --git a/arch/mips/netlogic/dts/xlp_fvp.dts b/arch/mips/netlogic/dts/xlp_fvp.dts
index 4aea53c..c8e4a2e 100644
--- a/arch/mips/netlogic/dts/xlp_fvp.dts
+++ b/arch/mips/netlogic/dts/xlp_fvp.dts
@@ -157,6 +157,18 @@
 				reg = <0x40000000 0x0>;
 			};
 		};
+		ethernet@3,0 {
+		        compatible = "smsc,lan9115";
+		        reg = <1 0x01200000 0x100000>;
+			reg-io-width = <4>;
+
+		        interrupts = <39>;
+		        interrupt-parent = <&gpio>;
+
+		        phy-mode = "mii";
+		        mac-address = [00 11 22 33 44 55];
+		        smsc,irq-push-pull;
+		};
 		spi: xlp_spi@3a000 {
 			compatible = "netlogic,xlp-spi";
 			#address-cells = <1>;
diff --git a/arch/mips/netlogic/dts/xlp_svp.dts b/arch/mips/netlogic/dts/xlp_svp.dts
index 04ee8f7..6710cf8 100644
--- a/arch/mips/netlogic/dts/xlp_svp.dts
+++ b/arch/mips/netlogic/dts/xlp_svp.dts
@@ -157,6 +157,18 @@
 				reg = <0x40000000 0x0>;
 			};
 		};
+		ethernet@3,0 {
+		        compatible = "smsc,lan9115";
+		        reg = <1 0x01200000 0x100000>;
+			reg-io-width = <4>;
+
+		        interrupts = <39>;
+		        interrupt-parent = <&gpio>;
+
+		        phy-mode = "mii";
+		        mac-address = [00 11 22 33 44 55];
+		        smsc,irq-push-pull;
+		};
 		spi: xlp_spi@3a000 {
 			compatible = "netlogic,xlp-spi";
 			#address-cells = <1>;
-- 
1.7.1

